WPMedia/fusion-news-theme-blocks
# TMEDIA-595 Add fetching next data set to collection API

## Description

Switch the collection API to use fetch to allow fetching the next "set" of results from a collection due to the limits of the collection API being set to maximum of 20

## Jira Ticket
- [TMEDIA-595](https://arcpublishing.atlassian.net/browse/TMEDIA-595)

## Acceptance Criteria
_copy from ticket_

## Test Steps

1. Checkout this branch `git checkout TMEDIA-595-collection-fetch-next`
2. Run `npm i` from the root of the blocks repo
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/collections-content-source-block,@wpmedia/results-list-block`
3. Using the results list block with collections set up, see more button is now working as expected when using any size value upto 20


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
